test/remote/gateways/remote_paymill_test.rb

```diff
@@ -2,71 +2,79 @@
 
 class RemotePaymillTest < Test::Unit::TestCase
   def setup
-    @gateway = PaymillGateway.new(fixtures(:paymill))
+    @gateway = PaymillGateway.new( fixtures(:paymill) )
 
     @amount = 100
     @credit_card = credit_card('5500000000000004')
     @declined_card = credit_card('5105105105105100', month: 5, year: 2020)
+
+    uri = URI.parse("https://test-token.paymill.com?transaction.mode=CONNECTOR_TEST&channel.id=#{@gateway.options[:public_key]}&jsonPFunction=paymilljstests&account.number=4111111111111111&account.expiry.month=12&account.expiry.year=2015&account.verification=123&account.holder=John%20Rambo&presentation.amount3D=#{@amount}&presentation.currency3D=EUR")
+    https = Net::HTTP.new(uri.host, uri.port)
+    https.use_ssl = true
+    request = Net::HTTP::Get.new(uri.request_uri)
+    request.basic_auth( ENV['PAYMILL_API_TEST_KEY'], "" )
```

Comment:
Can this come from fixtures as well? Right now an integrator who wants to run these tests won't be able to if they don't have a test key, even though there's probably one that will work in the fixtures file